File: app/src/main/java/com/sansu/aiforexbot/app/src/main/java/com/sansu/aiforexbot/MainActivity.java
package com.sansu.aiforexbot;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        TextView screen = new TextView(this);
        screen.setText("Sansu Real Forex Bot\n\nExness Demo\nXAUUSD\n\nBot: STOPPED");
        screen.setTextSize(22);

        setContentView(screen);
    }
}

File: app/src/main/java/com/sansu/aiforexbot/MainActivity.kt
package com.sansu.aiforexbot

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val screen = TextView(this)
        screen.text = "Sansu Real Forex Bot\n\nExness Demo\nXAUUSD\n\nBot: STOPPED"
        screen.textSize = 22f

        setContentView(screen)
    }
}
